doctr/datasets/iiit5k.py

```diff
@@ -65,22 +65,20 @@ def __init__(
             if not os.path.exists(os.path.join(tmp_root, _raw_path)):
                 raise FileNotFoundError(f"unable to locate {os.path.join(tmp_root, _raw_path)}")
 
-            # x, y, width, height -> xmin, ymin, xmax, ymax
-            box_targets = [[box[0], box[1], box[0] + box[2], box[1] + box[3]] for box in box_targets]
-
             if rotated_bbox:
                 # box_targets: xmin, ymin, xmax, ymax -> x, y, w, h, alpha = 0
                 box_targets = [
                     [
                         (box[0] + box[2]) / 2, (box[1] + box[3]) / 2, box[2] - box[0], box[3] - box[1], 0
```

Comment:
Careful here, you have to edit those so that it's still correct:

```python
box[0] + box[2] / 2, box[1] + box[3] / 2, box[2], box[3], 0
```

Reply:
@fg-mindee 
Oh yes it was a long time today sorry for this negligent mistake :see_no_evil:

Reply:
still not the good coordinates, cf my previous comment :upside_down_face:

Reply:
yes its only on width and height 
i´m out today have a nice evening :wave: